com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Drawers.cs

```diff
@@ -76,6 +78,9 @@ enum ShutterSpeedUnit
             new Vector2(70.41f, 52.63f)
         };
 
+        // Saves the value of the sensor size when the user switches from "custom" size to a preset.
+        static Vector2 s_LastCustomSensorSizeValue;
```

Comment:
There is a limitation in this fix: the memory for the old custom values when switching back from a preset is not per-camera but global (saved in a static variable). Otherwise we would have to serialize the old values, which sounds overkill!

Reply:
What happens if the following scenario incur: 

I have 2 cameras, A and B. 

I change value for camera A from 8mm to custom (values X,Y) 
Then I change camera B from 8mm to custom. 

Will camera B have the values from A? And if so, say I then change  the values on B as they are not what I want

If I change camera A to 8mm and then back to custom, I will now get the values from B?

Reply:
Yes, this is indeed the case and I agree it is not ideal (that's why I wrote the comment)

To clarify, we are talking about the default value that the user will see when selecting "custom" and before he writes his own preference. 
He can still have different custom size per camera as before, only the UI history that is common.

In practice, I don't believe there is a nice/clean way to fix this case: 
Each time you ping-pong between the cameras, the Editor is rebuild from scratch. Which means I would have to store the UI history in the run-time object (unless I'm missing something, please let me know in this case).

And storing any UI State in runtime does not sound good to me. So my personal opinion is that we could live with this specific limitation. 

If you or Seb insist, I can still put the "LastCustomSensorSizeValue" in HDPhysicalCamera.

Reply:
Ok, I think we should be good now, the ping-pong scenario that you describe now works as expected. In the end, I solved the per-camera history problem using a static dictionary (not sure why I did not think about this sooner)